web-ui/src/components/search-results/SearchBirthdayAnniversaryResults.jsx

```diff
@@ -15,26 +15,25 @@ import {
 
 const SearchBirthdayAnniversaryResults = ({
   hasSearched,
-  searchBirthdayResults,
```

Comment:
Instead of taking two arrays of results, this now only takes one because we no longer operate on both anniversaries AND birthdays.